app/models/chapter.rb

```diff
@@ -26,4 +26,8 @@ def index_usage!(organization = Organization.current)
     organization.index_usage_of! topic, self
     lessons.each { |lesson| lesson.index_usage! organization }
   end
+
+  def structural_parent
```

Comment:
With this new structure we can rethink the navigation flow. I think some improvements can be made to the *Navigation mixins and to the breadcrumb logic.
We can open a separate issue for that, wdyt @julian-berbel ?

Reply:
Agreed!